### ISSUE TITLE
The "Build for Solana" steps in readme.md does not work

### ISSUE THREAD

**Issue by swaglord8** [hyperledger/solang#1177]
Running `solang compile --target solana flipper.sol` generates flipper.abi & flipper.so, but not a flipper.json, which is used as IDL input for an anchor program later in the javascript integration example. 

Switching flipper.json for flipper.abi gives me other errors:
`TypeError: Cannot read properties of undefined (reading 'address')`

(IDL.metadata is not defined)

Compiling and deploying the contract works, but what do I need to do to correctly run the flipper.js example?

Versions used:
solang: v0.2.1
@project-serum/anchor: ^0.26.0
@solana/solidity: ^0.0.25
@solana/web3.js: "^1.73.2

**Comment by swaglord8** [comment]
Using @solana/solidity instead of Anchor works

**Comment by seanyoung** [comment]
@swaglord8 you are correct, this does not work. The last release of solang does not create the idl json file. The code on the latest main does this; we've been getting ready to do a release of solang. This release is currently planned for this Monday (the 20th).

In the mean time you can use the solang binary from our CI builds, like here for example: https://github.com/hyperledger/solang/actions/runs/4167680153

The `@solana/solidity` library is no longer supported and we will use anchor from now forward.

**Comment by harsox** [comment]
@seanyoung That worked! Thanks
